kolibri/core/content/api.py

```diff
@@ -610,7 +610,21 @@ def get_resume_queryset(request, queryset):
         .exclude(progress=1)
         .values_list("content_id", flat=True)
     )
-    return queryset.filter(content_id__in=content_ids)
+    queryset = queryset.filter(content_id__in=content_ids)
+    queryset = queryset.annotate(
```

Comment:
Realizing that now we are doing the ordering with the `OrderingFilter` we might want to move where we're doing this annotation, as this annotation will only be applied when we use the `resume` filter at the moment.